components/studio/bin/hab-studio.ps1

```diff
@@ -568,17 +572,22 @@ if(!(Test-InContainer)) {
 }
 
 try {
-    switch ($command) {
-        "new" { New-Studio }
-        "run" { Invoke-StudioRun $commandVal }
-        "rm" { Remove-Studio }
-        "enter" { Enter-Studio }
-        "build" { Invoke-StudioBuild $commandVal $R }
-        "version" { Write-Host "$program $version" }
-        "help" { Write-Help }
-        default {
-            Write-Help
-            Write-Error "Invalid Argument $command"
+    if ($extraParameters.Count -gt 0) {
```

Comment:
this can instead be `if ($args.count -gt 0) {`

In powershell you have 2 automatic variables for arguments `$args` and `$PSBoundPaeameters`. `$args` is the array of "unbound" arguments. So in this case `-d` would qualify.

Reply:
This is much better thanks. Addressed in [27e558](https://github.com/habitat-sh/habitat/pull/7585/commits/27e558c4c733781eefa76f70a428cf0d92d493fe)